### DISCUSSION TITLE
[FE101] 關於 <iframe> 用法的問題

### DISCUSSION THREAD

**Discussion by KW0409** [Lidemy/forum#124] [前端相關問題] [answered]
想知道 `<iframe>` 的用法是不是應該要有結束標籤
像這樣 `<iframe src='123123'></ iframe>`
因為在 [FE101] 的影片中，老師的用法是
 `<iframe src='123123' />`
但是我發現 `<iframe src='123123' />` 這樣的用法會造成內嵌網頁下面 html 的內容不見

**Comment by aszx87410** [comment, marked as answer]
對，應該是要有結束標籤的

**Reply by KW0409** [reply]
好的👌謝謝老師
